sdk/identity/azure-identity/tests/test_client_secret_credential_async.py

```diff
@@ -260,3 +260,8 @@ async def test_persistent_cache_multiple_clients():
     token_b = await credential_b.get_token(scope)
     assert token_b.token == access_token_b
     assert transport_b.send.call_count == 1
+
+@pytest.mark.asyncio
+async def test_token_refresh_offset():
+    credential = ClientSecretCredential("tenant-id", "client-id", "client-secret")
+    assert credential.token_refresh_offset == DEFAULT_REFRESH_OFFSET
```

Comment:
It would be more interesting to test whether the property returns a configured non-default value.